tests/queries/0_stateless/00062_replicated_merge_tree_alter_zookeeper.reference

```diff
@@ -1,22 +1,22 @@
 d	Date					
 k	UInt64					
 i32	Int32					
-CREATE TABLE default.replicated_alter1\n(\n    `d` Date,\n    `k` UInt64,\n    `i32` Int32\n)\nENGINE = ReplicatedMergeTree(\'/clickhouse/tables/test_00062/alter\', \'r1\', d, k, 8192)
+CREATE TABLE default.replicated_alter1\n(\n    `d` Date,\n    `k` UInt64,\n    `i32` Int32\n)\nENGINE = ReplicatedMergeTree(\'/clickhouse/tables/default/test_00062/alter\', \'r1\', d, k, 8192)
```

Comment:
Why we expect `default` database here?

Reply:
Because clickhouse-test normalizes stdout, and replace random database with default, see https://github.com/ClickHouse/ClickHouse/blob/5c3bd2980ec82ac85aa7d78c0a7df7e19bad59e4/tests/clickhouse-test#L264